pool-interface/src/lib.rs

```diff
@@ -177,7 +187,7 @@ pub trait LendingPoolTrait {
         to: Address,
     ) -> Result<(), Error>;
 
-    fn stoken_underlying_balance(env: Env, stoken_address: Address) -> i128;
+    fn stoken_underlying_balance(env: Env, asset: Address, stoken_address: Address) -> i128;
```

Comment:
Can we not change interface for this method? s_token_address can be retrieved from resere data